.tekton/tasks/echo-env.yaml

```diff
@@ -8,6 +8,12 @@ spec:
       value: $(params.node-version)
     - name: npm-version
       value: $(params.npm-version)
+    - name: prerelease
+      value: $(params.prerelease)   
+      default: ""
+    - name: prerelease-node-version
+      value: $(params.prerelease-node-version)   
+      default: ""
```

Comment:
No need to add defaults here.

Reply:
This change will also affect the tekton lint

Reply:
That might be, but we do not have to define defaults in the task :)
Please check